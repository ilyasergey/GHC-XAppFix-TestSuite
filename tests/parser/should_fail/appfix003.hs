-- Parsing the local 'alet' form

{-# LANGUAGE ApplicativeFix #-}

module ShouldCompile where

foo = alet a :: t -> t
           a = undefined
           b = undefined
        in a
