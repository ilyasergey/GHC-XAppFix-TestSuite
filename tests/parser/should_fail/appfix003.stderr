
appfix003.hs:7:12:
    No explicit type signatures allowed in alet: a :: t -> t
